operations/onboarding.md

```diff
@@ -0,0 +1,99 @@
+# Welcome to the TOC!
+
+We are so glad you are here. You have been elected to the Technical Oversight Committee, the body responsible for accepting and overseeing projects in the cloud native ecosystem.  This onboarding document and its various files and links are here to help get you up to speed and set you up for success as a TOC member. If you haven't already done so, now is a great time to familiarize yourself with the [Definition of Cloud Native](../DEFINITION.md).
```

Comment:
change to "the body responsible for reviewing, accepting/rejecting, and overseeing projects..."